code/game/shuttles/vox.dm

```diff
@@ -62,6 +62,10 @@ var/global/datum/shuttle/vox/vox_shuttle = new(starting_area=/area/shuttle/vox/s
 	link_to(vox_shuttle)
 	.=..()
 
+/obj/machinery/computer/shuttle_control/vox/attack_ai(mob/user)
+	to_chat(user, "<span class='warning'>This computer has a firewall preventing you from interacting with it.</span>")
```

Comment:
they understand the written language, but they don't understand the code obviously